backend/src/main/java/MultipleChoiceQuestion.java

```diff
@@ -0,0 +1,65 @@
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * A question type that allows selecting one option from a dropdown of many
+ */
+
+@Entity
+public class MultipleChoiceQuestion extends Question {
+
+
+    private List<String> options;
+    private List<String> answers;
```

Comment:
these need to be in a hashmap, should follow the uml we made earlier

Reply:
this will also change the other methods, because it should be a hashmap

Reply:
What I was gonna say about the hashmap implementation was that I didn't understand it so I implemented them as two seperate array lists

Reply:
so for the hashmap the keys of the map are the options and the key is an integer with the number of responses from that question. initially 0 then when we get more responses from the survey it increases.

Reply:
so based on the UML diagram we created we made , MultipleChoiceQuestion constructor takes a list of option, would I have to iterate through the hasp map and add those options from the list to the hashMap and set each value to zero?

Reply:
Also for the addAnswer method in this class would you like me to return true of false whether the option exists or not?

Reply:
yes iterate over the options , they default to 0. Also dont resolve the comments until youve actually made the changes

Reply:
I have made the changes

Reply:
then you need to commit and push the changes.
You need to familiarize yourself with git

Reply:
> Also for the addAnswer method in this class, I return true or false whether the option exists or not?

Reply:
for this one actually since it is a hashmap, we probably dont need to return a boolean for the addAnswer

Reply:
which is what the uml shows